resources/views/training/training.blade.php

```diff
@@ -0,0 +1,54 @@
+@extends('layouts.layout')
+
+@section('content')
+
+    <h1>Trainingen</h1>
+    @foreach($trainingGroups as $group)
+
```

Comment:
Het is niet een must, maar ik denk wel dat het mooiste is om links en rechts margin te hebben als op de andere paginas. Vooral voor de consistentie is dit het beste

Reply:
Dit valt overigens heel makkelijk op te lossen door om deze view een div te zetten met class container. Dus dan open je de div na @section en sluit je het voor de @stop

Reply:
Ooh, die div met container class zat eerst in layout. Die is echter weg gehaald wat ik niet door had. Fixed